test/boost/secondary_index_test.cc

```diff
@@ -41,6 +41,23 @@ SEASTAR_TEST_CASE(test_secondary_index_regular_column_query) {
         });
     });
 }
+// Reproduces #20772
```

Comment:
`#20722` -> `scylladb/scylladb#20722`
When the commit makes it to Enterprise, it may be ambiguous which issue we mean.